devs2025.yml

```
@@ -28,19 +28,172 @@ translations:
+
+  - key: technology.workplace.workplace_difficulties.takeaway.devs2025
+    t: |
+      Зрештою, всім нам потрібно заробляти на життя, і погана компенсація посіла перше місце як головна проблема робочого місця, з якою розробники стикаються сьогодні.
+
+  # technology
+
+  # - key: technology.mobile_os.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  # - key: technology.desktop_os.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  # - key: technology.programming_languages.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  - key: technology.productivity_apps.takeaway.devs2025
+    t: |
+      Коли йдеться про додатки для продуктивності, варто відзначити вражаючий результат **Obsidian**, який посів 6-те місце загалом, попри те, що навіть не був частиною попередньо визначених варіантів питання.
+
+      Деякі інструменти, такі як **Azure DevOps** та **Jira**, схильні до використання у великих компаніях, тоді як **Linear** та **Trello** популярніші серед менших організацій.
+
+      При розгляді виключно віддалених організацій,
+
+  - key: technology.chat_apps.takeaway.devs2025
+    t: |
+      Я настільки ж здивований, як і ви, дізнавшись, що **Google Chat** досі існує, проте він посів 9-те місце виключно завдяки відповідям, написаним від руки.
+
+  - key: technology.social_media.takeaway.devs2025
+    t: |
+      Mastodon та Bluesky можуть бути в авангарді нового контркультурного руху, але виявляється, що їхні користувачі також ті, кому потрібно розтягуватися щоразу, коли вони встають зі стільця, і які люблять розповідати про часи комутованого з'єднання та касетних стрічок.
+
+      TikTok, з іншого боку, схиляється до молодших користувачів, що має сенс, оскільки потрібне молоде тіло, щоб навіть *спробувати* деякі з тих танців.
+
+  - key: technology.community_involvement.takeaway.devs2025
+    t: |
+      Нічого дивного, але відео є основним способом, яким респонденти стежать за спільнотою. На жаль, особисті заходи залишаються досить рідкісними.
+
+      І хоча RSS-стрічки були популярною відповіддю, написаною від руки, медіанний вік цих респондентів не віщує добра для майбутнього цієї конкретної технології.
+
+  - key: technology.community_contributions.takeaway.devs2025
+    t: |
+      Приємно бачити, що розробники знаходять способи віддавати спільноті.
+
+  - key: technology.open_source_involvement.takeaway.devs2025
+    t: |
+      While almost everybody having used open-source software is expected, seeing that close to half of all respondents have contributed code back to open-source projects is a very encouraging sign.
+
+  # health
+  - key: technology.health.gender_vs_health_issues.takeaway.devs2025
+    t: |
+      Поганий сон був проблемою здоров'я, про яку респонденти повідомляли найчастіше, що може бути пов'язано з великою роллю екранів усіх видів у нашому житті.
+
+      Деякі проблеми, такі як проблеми психічного здоров'я та мігрені, також впливали на жінок і небінарних респондентів частіше порівняно із загальною популяцією.
```

```suggestion
      Деякі проблеми, як-от проблеми психічного здоров'я та мігрені, також впливали на жінок і небінарних респондентів частіше, як порівняти із загальною популяцією.
```
https://onlinecorrector.com.ua/%D1%8F%D0%BA-%D0%BF%D0%BE%D1%80%D1%96%D0%B2%D0%BD%D1%8F%D1%82%D0%B8-%D0%BF%D0%BE%D1%80%D1%96%D0%B2%D0%BD%D1%8E%D1%8E%D1%87%D0%B8-%D0%B7/